packages/moonstone/TooltipDecorator/util.js

```diff
@@ -0,0 +1,155 @@
+// TooltipDecorator util.js
+//
+
+/**
```

Comment:
I was going to comment that we shouldn't document these with jsdoc blocks since they aren't reexported... However, VS Code automatically parses these out and makes the docs and types available.  It might be a good idea to revisit some of the other places we disabled the doc blocks and turn them private for this enhanced capability.

Reply:
Should I take this out?

Reply:
no, no...  It's fine.